content/technical_challenges.tex

```diff
@@ -71,36 +71,136 @@ \subsection{Visual Referee Challenge}
+        \end{enumerate}
 
-        This procedure will be continued for in total four times. Each time, the referee chooses a new decision.
+        This procedure will be repeated \textit{five} times. Each time, the referee chooses a new decision. The referee should choose 2 one-hand static signals, 2 two-hand static signals, and 1 dynamic signal.
 
-    \subsubsection{Available Decisions}
+    \subsubsection{Available Hand-Signals}
 
-        For each decision (not all, because some decisions have to be shown on place, e.g., throw in) will be described and pictured.
+        Each hand-signal for the challenge is described and pictured. Note that for the purpose of clarity, these do not necessarily correspond to human soccer hand-signals.
 
         \begin{itemize}
-            \item \textbf{Kick-Off}
+            \item \textbf{Kick-in \textlangle{}colour\textrangle{} Team.}
+            One-handed signal. One arm, extended horizontally in the direction of the half of the field corresponding to the team that receives the Kick-in Free Kick. That is, right arm extended for the ``Blue team'', and left arm extended for the ``Red team''.
+            \todo{I'm not sure if this is consistent with human soccer, this could be inverted}.
+%            \begin{figure}[ht!]
+%                \includegraphics{figs/kick-off_referee.jpg}
+%            \end{figure}
+            \begin{figure}[ht!]
+                \centering
+                \begin{subfigure}{.33\textwidth}
+                  \includegraphics[height=130px]{figs/referee-signals/kick-in.png}
+                \end{subfigure}
+                \begin{subfigure}{.33\textwidth}
+                  \includegraphics[height=130px]{figs/referee-signals/kick-in-flipped.png}
+                \end{subfigure}
+            \end{figure}
+            
+            
+            \item \textbf{Goal Kick \textlangle{}colour\textrangle{} Team.}
+            One-handed signal. One arm, extended 45-degree \emph{up} in the direction of the end of the field where the goal kick will occur. That is, right arm extended for the ``Blue team'', and left arm extended for the ``Red team''.
+            \begin{figure}[ht!]
+                \centering
+                \begin{subfigure}{.33\textwidth}
+                  \includegraphics[height=130px]{figs/referee-signals/goal-kick.png}
+                \end{subfigure}
+                \begin{subfigure}{.33\textwidth}
+                  \includegraphics[height=130px]{figs/referee-signals/goal-kick-flipped.png}
+                \end{subfigure}
+            \end{figure}
+
+
+            \item \textbf{Corner Kick \textlangle{}colour\textrangle{} Team.}
+            One-handed signal. One arm, extended 45-degree \emph{down} in the direction of the end of the field where the corner kick will occur. That is, right arm extended for the ``Blue team'', and left arm extended for the ``Red team''.
```

Comment:
Aren't here the teams reversed. Because corner kick happens for the red on the right half and for blue on the left half

Reply:
Ah yes. These should be interchanged.